test/linalg/bidiag.jl

```diff
@@ -97,29 +97,30 @@ srand(1)
         end
 
         @testset "triu and tril" begin
+            bidiagcopy(dv, ev, uplo) = Bidiagonal(copy(dv), copy(ev), uplo)
```

Comment:
No but previously the constructor always made a copy of the vectors, ref https://github.com/JuliaLang/julia/commit/e0d75d7e998cf04e6634f4b92c6d5dbc2776dcb5#commitcomment-22968553